docs/source/_templates/_static/css/ignite_theme.css

```diff
@@ -168,3 +168,7 @@ footer table {
   width: 100%;
   margin-bottom: 1rem;
 }
+
+.pytorch-body .docs-tutorials-resources {
+  padding: 0 !important;
```

Comment:
Can you please explain which gray footer this removes ?

Reply:
It removes this gray footer
![Screen Shot 2021-03-02 at 12 57 42](https://user-images.githubusercontent.com/32727188/109607668-f7d0eb00-7b56-11eb-8f00-59b78744eb82.png)

Reply:
Thanks for the screenshot. Just to make sure, we do not see it currently neither on https://pytorch.org/ignite/ right ?

Reply:
Yes, but if you see the left side bar when you scroll to the end of the engine page, the left side bar disappears with scrolling. To have the left side bar to stick, we need to enable `<div class="container-fluid docs-tutorials-resources" id="docs-tutorials-resources">` this line. When this line is enabled, that gray footer appears, so we need padding 0 around them.

Reply:
Sounds good! Thanks for the explanation!